README.md

```diff
@@ -1,654 +1,43 @@
-
-In order to track the host shutdown progress, run the following command:
-```
-kubectl logs pod/terminated-host-pod-name
-```
-
-If you are permanently removing the host from the MarkLogic cluster, once the pod is terminated, follow standard MarkLogic administrative procedures using the administrative UI or APIs to remove the MarkLogic host from the cluster. Also, because Kubernetes keeps the Persistent Volume Claims and Persistent Volumes around until they are explicitly deleted, you must manually delete them using the Kubernetes APIs before attempting to scale the hosts in the StatefulSet back up again.
-### Enabling SSL over XDQP
-
-To enable SSL over XDQP, set the `enableXdqpSsl` to true either in the values.yaml file or using the `--set` flag. All communications to and from hosts in the cluster will be secured. When this setting is on, default SSL certificates will be used for XDQP encryption.
-
-Note: To enable other XDQP/SSL settings like `xdqp ssl allow sslv3`, `xdqp ssl allow tls`, `xdqp ssl ciphers`, use MarkLogic REST Management API. See the MarkLogic documentation [here](https://docs.marklogic.com/REST/management).
-
-# Deploying a MarkLogic Cluster with Multiple Groups
-
-To deploy a MarkLogic cluster with multiple groups (separate E and D nodes for example) the `bootstrapHostName` and `group.name` must be configured in values.yaml or set the values provided for these configurations using the `--set` flag while installing helm charts.
-For example, if you want to create a MarkLogic cluster with three nodes in a "dnode" group and two nodes in an "enode" group, start with the following helm command:
-
-```
-helm install dnode-group ./charts/ --set group.name=dnode --set replicaCount=3
-```
-Once this deployment is complete, a MarkLogic cluster with three hosts should be running.
-To add the "enode" group and nodes to the cluster, the `bootstrapHostName` must be set to join the existing MarkLogic cluster. The first host in the other group can be used. For this example, set `bootstrapHostName` to `dnode-group-marklogic-0.dnode-group-marklogic-headless.default.svc.cluster.local` with the following command:
-
-```
-helm install enode-group ./charts/ --set group.name=enode --set replicaCount=2 --set bootstrapHostName=dnode-group-marklogic-0.dnode-group-marklogic-headless.default.svc.cluster.local
-```
-Once this deployment is complete, there will be a new "enode" group with two hosts in the MarkLogic cluster.
-
-# Access the MarkLogic Server
-
-## Service
-
-You can use the ClusterIP service to access MarkLogic within the Kubernetes cluster.
-
-### Get the ClusterIP Service Name
+This helm chart installation will create a single node MarkLogic cluster with a Default group and a persistent volume of 10Gi attached to the pod created.
```

Comment:
Move this to be the first paragraph in the "Install MarkLogic Helm Chart" section. Also add more details about what gets allocated. CPU, RAM, etc.

Reply:
What will the admin user be? What will the admin password be? We need to list what those are.

Reply:
How would the user then access MarkLogic? We need to give at least a simple method to access the admin UI. Perhaps port forward with kubectl?

Reply:
added details